app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,17 @@
+class GuestsController < ApplicationController 
+
+  def show 
+    @guest = Guest.find(params[:id])
+    @guest_rooms = Guest.find(params[:id]).rooms.first
+  end
+
+  def create 
+    @guest = Guest.find(params[:id])
+    @new_room = Room.find(params[:room_id])
+
+    @guest.rooms << @new_room
+    @guest.save
+
+    redirect_to "/guests/#{@guest.id}"
+  end
```

Comment:
I like that this logic is in a `create` method, but what kind of resource are we creating? We're actually creating a GuestRoom object (on line 12, you're creating that association, so that's where that resource is getting created). For that reason, it would be most ReSTful to have this in a `GuestRoomsController` in a `create` action. 

Also, we could make this code a bit more optimal. Right now, you're finding the guest and finding the room, and then creating the object. But, you actually have EVERYTHING you need to make that connection without first finding those two objects. You could do the following: 

```ruby
GuestRoom.create(guest_id: params[:id], room_id: params[:room_id])
redirect_to "/guests/#{params[:id]}"
```